linera-execution/src/runtime.rs

```diff
@@ -1612,6 +1622,7 @@ impl ServiceSyncRuntime {
                 None,
                 ResourceController::default(),
                 (),
+                false, // Services don't output contract logs
```

Comment:
Services should be able to output logs. This is actually less controversial because it often runs on the client.